bagitobjecttransfer/recordtransfer/storage.py

```diff
@@ -1,25 +1,53 @@
+from pathlib import Path
```

Comment:
Need urljoin for a change I highlighted below.

```suggestion
from pathlib import Path
from urllib.parse import urljoin
```

Reply:
Using `as_posix()` on `base_url` `Path` instead of using `url_join`

Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/292/commits/152ecf548e49cc79804dcac70cb6b3d58491026b